libs/sdk-ui-dashboard/src/presentation/filterBar/utils.ts

```diff
@@ -1,8 +1,20 @@
 // (C) 2023 GoodData Corporation
 import { IntlShape } from "react-intl";
-import { messages } from "../../locales.js";
+
```

Comment:
Please help to remove this empty line :) we don't need to separate the import libraries.

Reply:
done!